docs/src/user/manual/help.md

```diff
@@ -0,0 +1,82 @@
+#context_id: 1234
+---
+
+# Help
+
+[![Home Page](./img/help-img-3.png)](./img/help-img-3.png)
+
+
+## Getting help using the FBIS platform
+
+We are so pleased that users are using FBIS! The site offers a broad range of features, which are documented in the user [guide](../guide/index.md) and [manual](./index.md). For a quick start, users can explore the walk-through video of the platform. Lastly, although we strive to provide a seamless experience, users may occasionally encounter issues with the site. If such an issue arises, users are encouraged to report it on our issue tracker, and we will do our best to resolve it promptly!
+
+## FBIS Testing site
+
+For practicing the workflows for capturing data on FBIS, users should use the [FBIS Testing site](https://testing.healthyrivers.kartoza.com/). Here, users can add dummy data, as the data are overwritten with a mirror image of the FBIS Live (production) site every week. The same login works for both `FBIS Live` and `FBIS Testing`.
+
+## User Manual
+
+User can also checkout the [user manual](https://freshwaterbiodiversity.org/uploaded/documents/document/MmWwDIn-SCmrqt_lyQVbVw.pdf) for more information about the platform.
+
+## instructional Videos
+
+We have also created a series of instructional videos to help users get started with the platform.
+
+* **Introduction to key FBIS components:**
+
+    [![Dummy Image](./img/help-img-2.png)](https://www.youtube.com/watch?v=aex5tzLNWAo)
+
+* **Filtering data:**
+
+    [![Dummy Image](./img/help-img-2.png)](https://www.youtube.com/watch?v=ygVj7yVYyQE)
+
+* **Overview of spatial layers:**
+
+    [![Dummy Image](./img/help-img-2.png)](https://www.youtube.com/watch?v=_AtjWcbYLWc)
+
+* **Single and multi-site dashboards:**
+
+    [![Dummy Image](./img/help-img-2.png)](https://www.youtube.com/watch?v=NTGyonmYO9Y)
+
+* **Single and multi-site SASS dashboards:**
+
+    [![Dummy Image](./img/help-img-2.png)](https://www.youtube.com/watch?v=wPSNDMAyCY4)
+
+* **Taxon dashboard:**
+
+    [![Dummy Image](./img/help-img-2.png)](https://www.youtube.com/watch?v=mT152Z-Dc4s&feature=youtu.be)
+
+*  **Exporting graphs and CSV files:**
+
+    [![Dummy Image](./img/help-img-2.png)](https://www.youtube.com/watch?v=xxyER7wa1ts)
+
+* **Adding a site:**
+
+    [![Dummy Image](./img/help-img-2.png)](https://www.youtube.com/watch?v=RzfL8P7prag)
+
+* **Adding biodiversity data:**
+
+    [![Dummy Image](./img/help-img-2.png)](https://www.youtube.com/watch?v=1rZByIjWZF0)
+
+* **Adding SASS data:**
+
+    [![ Dummy Image](./img/help-img-2.png)](https://www.youtube.com/watch?v=9JI2D1c-1Jk)
+
+*  **Preparing and adding water temperature time series data:**
+
+    [![Dummy Image](./img/help-img-2.png)](https://www.youtube.com/watch?v=9JI2D1c-1Jk)
+
+* **Preparing an occurrence data upload file:**
+
+    [![Dummy Image](./img/help-img-2.png)](https://www.youtube.com/watch?v=hWoFoPBdWI4)
```

Comment:
Please embed the youtube videos using the mkdocs-videos plugin.

Reply:
I have done it in my previous PR 
This branch is made from the user manual branch that's why it is showing here as well.
I will rebase this branch from the main, resolve conflicts, and prepare this PR for review.